### ISSUE TITLE
Provide mechanism to invalidate call caching for workflow

### ISSUE THREAD

**Issue by geoffjentry** [broadinstitute/cromwell#1670]
I would like to be able to tell Cromwell to not use a workflow for cached results without mucking around the database. 
![628b747f8ccdfb757062f36a27eedecfc2295f515c0586e05fbfb0620c0571a2](https://cloud.githubusercontent.com/assets/961771/20325706/cb81eed2-ab53-11e6-90a5-802160419fa5.jpg)

**Comment by katevoss** [comment]
@geoffjentry what is the current state of things, does Cromwell "muck" around the database?

**Comment by geoffjentry** [comment]
Yeah, and this just came up recently I know. But it might have been from your archaeology adventures.

This really shouldn't have the `technical debt` label on it, this is just missing functionality.

**Comment by katevoss** [comment]
What's the functionality that's missing?

**Comment by geoffjentry** [comment]
There's currently no mechanism to tell cromwell to not use an old workflow for call caching purposes. It's possible that I realize that something about it was bad or maybe I just don't want to use it in the future. 

I have no idea why I cared 9 months ago but I invoked Jean-Luc so it must have been good.

**Comment by geoffjentry** [comment]
I suspect what happened was someone was asking me to do it for them and the only way was to do it via the DB.

**Comment by katevoss** [comment]
I see, so in order to call cache on an old workflow Cromwell has to dig into the database. 

- Effort: **?**
- Risk: **Medium**
- Business value: **Small**

**Comment by geoffjentry** [comment]
@katevoss cromwell would have to regardless. i mean right now a human has to go into the DB

**Comment by katevoss** [comment]
As a **workflow runner**, I want **to selectively invalidate a workflow so that Cromwell does not use it for a cache-hit**, so that I can **not use bad or old workflow results in my new workflows**.
- Effort: **Medium**
- Risk: **Medium**
- Business value: **Small**
